src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -91,5 +101,14 @@ export default compose(
         personalDetails: {
             key: ONYXKEYS.PERSONAL_DETAILS_LIST,
         },
+        policy: {
+            key: ({policyID}) => `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+        },
     }),
-)(ReportActionItemCreated);
+)(
+    memo(
+        ReportActionItemCreated,
+        (prevProps, nextProps) =>
+            lodashGet(prevProps.props, 'policy.name') === lodashGet(nextProps, 'policy.name') && lodashGet(prevProps.props, 'policy.avatar') === lodashGet(nextProps, 'policy.avatar'),
+    ),
+);
```

Comment:
@hungvu193 I don't think we can make this change. There might be other multiple thing that should trigger the rerender like change in name of report or user or user's profile. Those should also be incorporated here.

Reply:
Yup.